stack/source.go

```diff
@@ -53,7 +53,7 @@ func (c *cache) augmentGoroutine(goroutine *Goroutine) {
 	}
 
 	// Once all loaded, we can look at the next call when available.
-	for i := 1; i < len(goroutine.Stack.Calls); i++ {
+	for i := 0; i < len(goroutine.Stack.Calls); i++ {
```

Comment:
IIRC, it is because the very first call is either main() or a goroutine with "created by foo".

In particular, line 321 in stack.go says "first is original function, last is leaf function". The first call is never "processable". Makes sense?

Reply:
Yeah, thanks, that explanation makes sense. I think the implementation was incorrect though. It appears the leaf call was the one being skipped instead of the first. I've included an example in the message in 400f29b3e7b9ab7bd946877fe44e8f08767ef058.